lib/python/src/bailo/core/agent.py

```diff
@@ -64,19 +64,14 @@ def put(self, *args, **kwargs):
 
 
 class PkiAgent(Agent):
-    def __init__(
-        self,
-        cert: str,
-        key: str,
-        auth: str,
-    ):
+    def __init__(self, cert: str, key: str, auth: str, **kwargs):
```

Comment:
@PE39806 I have run black which is collapsing this line now. is this an issue?

Reply:
That is fine. I think that, at some point, black's config has been modified and this is the first time this file has been changed since then.